diff -uNr doublecmd-0.8.2.orig/src/fmain.pas doublecmd-0.8.2/src/fmain.pas
--- doublecmd-0.8.2.orig/src/fmain.pas	2018-06-03 00:26:09.969714141 +0300
+++ doublecmd-0.8.2/src/fmain.pas	2018-06-03 00:26:18.312481757 +0300
@@ -3629,7 +3629,7 @@
   begin
     DisableAutoSizing;
     try
-      ScaleFontsPPI(AYProportion);
+      ScaleFontsPPI(0, AYProportion);
       BorderSpacing.AutoAdjustLayout(AXProportion, AYProportion);
       Constraints.AutoAdjustLayout(AXProportion, AYProportion);
     finally
